java/src/main/java/ai/rapids/cudf/Aggregation.java

```diff
@@ -49,8 +49,10 @@
         NTH_ELEMENT(16),
         ROW_NUMBER(17),
         COLLECT(18),
-        PTX(19),
-        CUDA(20);
+        LEAD(19),
```

Comment:
Are these the correct IDs?  My counting of enums in aggregation.hpp ends up with these being 20 and 21, respectively.  The Java list seems off starting after `COUNT` because the C++ header has two values instead, `COUNT_VALID` and `COUNT_ALL`.

I guess it works because the JNI code doesn't use the `enum Kind` from C++ but hardcodes the values, but it is quite confusing to follow since it's sorta trying to follow that enum without actually being that enum.